src/model_construction/construct_nodes.py

```diff
@@ -130,9 +130,15 @@ def init_carrier_parameter(para, t, car):
             """Rule initiating a carrier parameter"""
             return ts[car][key][t - 1]
 
-        parameter = Param(
-            set_t, b_node.set_carriers, rule=init_carrier_parameter, mutable=False
-        )
+        if key in ["Import emission factor", "Export emission factor"]:
```

Comment:
Also: I would suggest to make this more general: we can pass an optional argument to create_carrier_parameter, i.e. mutable_param = False. Then when this function is called you pass True instead for some of the parameters if monte carlo is on. So:

if monte_carlo_on:
mutable_param_on = True
else:
mutable_param_on = False

...

param = create_carrier_parameter(key, mutable_param=mutable_param_on)